automatic/Ghostscript.app/Ghostscript.app.nuspec

```diff
@@ -1,16 +1,23 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!-- Do not remove this test for UTF-8: if “Ω” doesn’t appear as greek uppercase omega letter enclosed in quotation marks, you should use an editor that supports UTF-8, not this one. -->
-<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
-    <id>{{PackageName}}</id>
-    <version>{{PackageVersion}}</version>
-    <title>Ghostscript (Install)</title>
+    <id>Ghostscript.app</id>
```

Comment:
we could always set the `PackageName` in the update.ps1 file.
But if we do, au won't be able to commit the files with `git` because of https://github.com/majkinetor/au/issues/66 and because `git` is case sensitive.